web/packages/design/src/SyncStamp/SyncStamp.story.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * Teleport
+ * Copyright (C) 2023  Gravitational, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025  Gravitational, Inc.
```

nit (and all other files too)